src/Yasumi/Provider/Argentina.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+declare(strict_types=1);
+/*
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2021 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Nader Safadi <nader@gizra.com>
```

Comment:
Please leave the original author name in this file comment. You can add your own name in the class comment.